man/mustache.5.ron

```diff
@@ -38,6 +38,13 @@ clauses, or for loops. Instead there are only tags. Some tags are
 replaced with a value, some nothing, and others a series of
 values. This document explains the different types of Mustache tags.
 
+The Mustache language has a [formal specification][spec]. The current manpage
+reflects version 1.1.3 of the specification, as well as the de facto
+established [candidate extension for template inheritance][inheritance].
```

Comment:
```suggestion
established [candidate extensions for template inheritance][inheritance] and lambdas.
```

Reply:
Well, I need to change this wording. At the time of writing, the status of the inheritance spec was different from the lambdas spec (not official vs official). I might now change it into just "including the official extensions".